barretenberg/cpp/src/barretenberg/commitment_schemes/gemini/gemini.hpp

```diff
@@ -239,12 +241,13 @@ template <typename Curve> class GeminiVerifier_ {
      * @param fold_polynomial_evals  Evaluations \f$ A_{i-1}(-r^{2^{i-1}}) \f$.
      * @return Evaluation \f$ A_0(r) \f$.
      */
-    static Fr compute_gemini_batched_univariate_evaluation(Fr& batched_eval_accumulator,
+    static Fr compute_gemini_batched_univariate_evaluation(size_t evaluation_point_size,
```

Comment:
please change to const size_t num_variables and remove the line below

Reply:
there will be follow ups to this where I put the const proof size in place and will clean up the function signatures more :)